src/desktop/apps/auction/queries/worksByFollowedArtists.js

```diff
@@ -56,6 +56,17 @@ export const worksByFollowedArtists = `
           }
           href
           is_acquireable
+          sale_artwork {
+            counts {
+              bidder_positions
```

Comment:
I see this `sale_artwork.counts.bidder_positions` attribute used in the code above, but where are we using the other fetched attributes like `sale.is_auction`, `sale.is_closed`, `sale_artwork.highest_bid.display`, and `sale_artwork.opening_bid.display`? Where we using `is_in_auction` before?

Reply:
`sale.is_auction`, `sale.is_closed`, `sale_artwork.highest_bid.display` used in the [Details component](https://github.com/artsy/force/blob/dimatretyak/feat/show-bid-on-mobile/src/v2/Components/Artwork/Details.tsx#L179-L198)

Reply:
Ah, so that component wasn't able to render correctly because we weren't fetching the appropriate data. Got it. 👍🏼

Reply:
Yes, you understood everything correctly 🥇